qiskit/visualization/circuit_visualization.py

```diff
@@ -46,14 +46,34 @@
 from qiskit.visualization import utils
 from qiskit.visualization import matplotlib as _matplotlib
 
-try:
-    subprocess.run("pdflatex -v", check=True)
-except OSError as ex:
-    HAS_PDFLATEX = False
 
 logger = logging.getLogger(__name__)
 
 
+class HasPdfLatexWrapper:
+    """Wrapper to lazily detect presence of the pdflatex command."""
+
+    has_pdflatex = False
+
+    # pylint: disable=unused-import
+    def __bool__(self):
+        if not self.has_pdflatex:
```

Comment:
This will re-run the check every time the wrapper is queried (and the result is `False`).  We could do
```suggestion
    has_pdflatex = None

    def __bool__(self):
        if self.has_pdflatex is None:
```
to avoid that, and remove the pylint comment, since there won't be a warning generated here.

Reply:
Fixed, pylint is from a quick copy-paste, sorry.